scripts/Run-Self-Hosted-Runner-Test.ps1

```diff
@@ -0,0 +1,265 @@
+# Copyright (c) Microsoft Corporation
+# SPDX-License-Identifier: MIT
+#
+# This script executes the provided test command, waits for <TestHangTimeout> (in seconds) and then triggers user and
+# kernel dumps if the test process is still ruunning (This is typically the case when the test process is hung for some
+# reason).
+#
+# The user mode dump is created using 'procdump64.exe' and the kernel dump using 'notmyfault64.exe' utilities
```

Comment:
Why not livedump64.exe?

Reply:
A live dump would be useful if the hang was in the user mode components of the test.  OTOH, if the hang is in the ebpf kernel components, there's no point in continuing further tests.

I guess determining the location of the hang _might_ be possible, I'm not sure what this additional complexity buys us in our current ci/cd scenario.